CMakeLists.txt

```diff
@@ -42,6 +42,8 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 option(alpaka_BUILD_EXAMPLES "Build the examples" OFF)
 
 option(BUILD_TESTING "Build the testing tree." OFF)
+
+option(alpaka_INSTALL_TEST_HEADER "Install headers of the namespace alpaka::test. Attention, headers are not designed for productive code, see documentation." OFF)
```

Comment:
I think you meant `production`, not `productive`

Reply:
If you change, the same line is present in `cmake.rst`

Reply:
Yes your are right. It makes more sense. I fixed it.